jax/experimental/jax2tf/primitives_with_limited_support.md

```diff
@@ -18,79 +18,50 @@ cases even for JAX, e.g., sort does not work for complex numbers on TPUs.
-| sinh | Missing TF support | Primitive is unimplemented for dtype float16 | CPU, GPU, TPU |
-| sort | Missing TF support | Primitive is unimplemented for dtype bool; sorting 2 arrays where the first one is an array of booleans is not supported for XlaSort | CPU, GPU, TPU |
-| sort | Missing TF support | Primitive is unimplemented for dtype complex128 | CPU, GPU, TPU |
-| sort | Missing TF support | Primitive is unimplemented for dtype complex64 | CPU, GPU, TPU |
-| sort | Missing TF support | Primitive is unimplemented; only sorting on last dimension is supported for XlaSort | CPU, GPU, TPU |
-| sort | Missing TF support | Primitive is unimplemented; sorting more than 2 arrays is not supported for XlaSort | CPU, GPU, TPU |
-| sort | Missing TF support | Primitive is unimplemented; stable sort not implemented for XlaSort | CPU, GPU, TPU |
-| svd | Missing TF support | Primitive is unimplemented for dtype complex128; this works on JAX because JAX uses a custom implementation | CPU, GPU |
-| svd | Missing TF support | Primitive is unimplemented for dtype complex64; this works on JAX because JAX uses a custom implementation | CPU, GPU |
-| top_k | Missing TF support | Primitive is unimplemented for dtype float64; this is a problem only in compiled mode (experimental_compile=True)) | CPU, GPU, TPU |
-| top_k | Missing TF support | Primitive is unimplemented for dtype int64; this is a problem only in compiled mode (experimental_compile=True)) | CPU, GPU, TPU |
-| top_k | Missing TF support | Primitive is unimplemented for dtype uint64; this is a problem only in compiled mode (experimental_compile=True)) | CPU, GPU, TPU |
+| Affected primitive | Type of limitation | Description | Affected dtypes | Affected devices |
+| --- | --- | --- | --- | --- |
+| acosh | Missing TF support | Primitive is unimplemented | bfloat16 | CPU, GPU |
+| acosh | Missing TF support | Primitive is unimplemented | float16 | CPU, GPU, TPU |
+| add | Missing TF support | Primitive is unimplemented | uint16, uint32, uint64 | CPU, GPU, TPU |
+| asinh | Missing TF support | Primitive is unimplemented | bfloat16 | CPU, GPU |
+| asinh | Missing TF support | Primitive is unimplemented | float16 | CPU, GPU, TPU |
+| atan2 | Missing TF support | Primitive is unimplemented | bfloat16, float16 | CPU, GPU, TPU |
+| atanh | Missing TF support | Primitive is unimplemented | bfloat16 | CPU, GPU |
+| atanh | Missing TF support | Primitive is unimplemented | float16 | CPU, GPU, TPU |
+| bessel_i0e | Missing TF support | Primitive is unimplemented | bfloat16 | CPU, GPU |
+| bessel_i1e | Missing TF support | Primitive is unimplemented | bfloat16 | CPU, GPU |
+| conv_general_dilated | Missing TF support | Primitive is unimplemented; likely bug in the HLO -> LLVM IR lowering of XlaConv | complex64, complex128 | CPU, GPU, TPU |
+| conv_general_dilated | Missing TF support | Primitive is unimplemented; batch_group_count != 1 unsupported |  | CPU, GPU, TPU |
```

Comment:
Perhaps here it would be better to show "ALL" instead of an empty list of affected dtypes.